e2e/utils/create-project-utils.ts

```diff
@@ -252,14 +252,15 @@ export function packageInstall(
 
 export function runNgNew(
   projectName: string,
-  packageManager = getSelectedPackageManager()
+  packageManager = getSelectedPackageManager(),
+  angularVersion = angularCliVersion
```

Comment:
Why not keep the same name - `angularCliVersion`?

Reply:
I have to rename one either way. But I went ahead and renamed the import instead.